testfiles/CoilWaterDesuperheatingUnitaryCoilDX.idf

```diff
@@ -0,0 +1,1690 @@
+!-Generator IDFEditor 1.34
```

Comment:
This file converts `CoilWaterDesuperheatingUnitary` from using the old single-speed coil to using the new `Coil:Cooling:DX` coil. This is what I've been comparing against the baseline.